docs/DeveloperGuide.md

```diff
@@ -257,33 +257,46 @@ _{Explain here how the data archiving feature will be implemented}_
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority | As a …​                                    | I want to …​                  | So that I can…​                                                       |
+|----------|--------------------------------------------|-------------------------------|-----------------------------------------------------------------------|
+| `* * *`  | new user                                   | see usage instructions        | refer to instructions when I forget how to use the App                |
+| `* * *`  | user                                       | add a new client              |                                                                       |
+| `* * *`  | user                                       | delete a client               | remove entries that I no longer need                                  |
+| `* * *`  | user                                       | find a client by name         | locate details of clients without having to go through the entire list|
+| `* * *`  | user                                       | save written information      | access previously saved information                                   |
+| `* *`    | user                                       | find a client by address      | locate details of clients in a designated area                        |
+| `* *`    | user                                       | have a schedule reminder      | keep track of when and which clients I am meeting                     |
+| `* *`    | tech-savvy user                            | input shortcut commands       | be more efficient using the app                                       |
+| `* *`    | user                                       | update a client's information | make changes whenever clients' information update                     |
+| `*`      | user with many persons in the client book  | sort clients by name          | locate a client easily                                                |
```

Comment:
Nice

"find a client by address": good point, we should prob have this feature. I'll add to UG